plugins/node/opentelemetry-instrumentation-cassandra/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@opentelemetry/instrumentation-cassandra-driver",
+  "version": "0.20.0",
+  "description": "OpenTelemetry instrumentation for cassandra-driver",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js-contrib",
+  "scripts": {
+    "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.test.ts'",
+    "tdd": "npm run test -- --watch-extensions ts --watch",
+    "clean": "rimraf build/*",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../",
+    "precompile": "tsc --version",
+    "prepare": "npm run compile",
+    "version:update": "node ../../../scripts/version-update.js",
+    "compile": "npm run version:update && tsc -p ."
+  },
+  "keywords": [
+    "opentelemetry",
+    "logging",
+    "nodejs",
+    "tracing",
+    "profiling",
+    "instrumentation",
+    "cassandra-driver"
+  ],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=8.5.0"
+  },
+  "files": [
+    "build/src/**/*.js",
+    "build/src/**/*.d.ts",
+    "doc",
+    "LICENSE",
+    "README.md"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "peerDependencies": {
+    "@opentelemetry/api": "^0.21.0"
+  },
+  "devDependencies": {
+    "@opentelemetry/api": "0.21.0",
```

Comment:
you can directly put api 1.0 (PR to update contrib https://github.com/open-telemetry/opentelemetry-js-contrib/pull/540 will likely be merged next week)

Reply:
Upgraded to 1.0.0